### PR TITLE
Add right margin to avatar group so other components don't overlap

### PR DESCRIPTION
**Pull request by petemcfarlane**

### CODE REVIEW

File: lib/AvatarGroup/styles.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.avatar-group-list {
+  margin-right: 1em;
```
Comment (kyleharper) [inline lib/AvatarGroup/styles.scss]: Ideal we'd use a spacing setting e.g. `$layout-spacing-base` but merge away as this will be cleaned up in another PR around Avatars.

Comment (petemcfarlane) [inline lib/AvatarGroup/styles.scss]: The reason I used `1em` is because that is the value that we loose here https://github.com/gathercontent/gather-ui/blob/63a5085884ac582479dc048b0189033639108c3c/lib/Avatar/styles.scss#L93-L99
when the avatars are in a group - this doesn't add any extra margin around the group, it just resets the margin that is lost, if that makes sense 😄

Comment (kyleharper) [inline lib/AvatarGroup/styles.scss]: Bah, something else that'll be resolved the #81 so 👍  Cheers dude ✋
